rocAL/source/api/rocal_api_data_loaders.cpp

```diff
@@ -2125,17 +2127,7 @@ rocalAudioFileSourceSingleShard(
         output = context->master_graph->create_loader_output_tensor(info);
         output->reset_audio_sample_rate();
         auto cpu_num_threads = context->master_graph->calculate_cpu_num_threads(shard_count);
-        context->master_graph->add_node<AudioLoaderSingleShardNode>({}, {output})->init(shard_id, shard_count, cpu_num_threads,
-                                                                                        source_path,
-                                                                                        "",
-                                                                                        StorageType::FILE_SYSTEM,
-                                                                                        DecoderType::SNDFILE,
-                                                                                        shuffle,
-                                                                                        loop,
-                                                                                        context->user_batch_size(),
-                                                                                        context->master_graph->mem_type(),
-                                                                                        context->master_graph->meta_data_reader()
-                                                                                        );
+        context->master_graph->add_node<AudioLoaderSingleShardNode>({}, {output})->init(shard_id, shard_count, cpu_num_threads, source_path, source_file_list_path, storage_type, DecoderType::SNDFILE, shuffle, loop, context->user_batch_size(), context->master_graph->mem_type(), context->master_graph->meta_data_reader());
```

Comment:
@SundarRajan28 
The storage_type has now been removed. 

As discussed already, 
Please handle this where we use the storage _type as file_list reader directly. As file source reader will not be able to handle the structure of folder/ sub_folder1 / inner_sub_folders.